calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
```

Comment:
Have you considered making a Calculator class that takes in the operator as an argument in its constructor function? You could even take some of the code from index.js and make methods where the Calculator prompts the user for input and then assigns those values to instance properties.